schemas/yazi.json

```diff
@@ -53,6 +54,7 @@
 				"tab_size": { "$ref": "/schemas/types/u8.json" },
 				"max_width": { "$ref": "/schemas/types/u32.json" },
 				"max_height": { "$ref": "/schemas/types/u32.json" },
+				"image_delay": { "$ref": "/schemas/types/u8.json" },
```

Comment:
Nitpicky but this should go below `cache_dir` and above `image_filter`, as is the order in Yazi's source code and preset theme files.

Reply:
This also is missing the minimum (of 0) and maximum (of 100) contraints.

Reply:
Now i moved `image_filter` down blow `cache_dir ` with new a value
```
	"image_delay": {"type": "number", "minimum": 0, "maximum": 100 },
```
which limits the value from minimum 0 to maximum 100. I totally agree with you that the order should be preserved.